### ISSUE TITLE
Downloaded extension is not deleted from /tmp folder

### ISSUE THREAD

**Issue by maroony** [microsoft/vscode-remote-release#6906]
This issue is related to https://github.com/Microsoft/vscode/issues/53110. 

Downloaded extension files are not deleted from `/tmp`. So this folder is getting bigger and bigger over time. Because the related issue is fixed for years now, maybe this issue occurs only when using remote-ssh extension.

- VSCode Version: 1.68.1
- Remote Extension/Connection Type: SSH

**Comment by sandy081** [comment]
Can you please let us know what files are there in the /tmp file?

**Comment by maroony** [comment]
It's exactly like in the linked issue. These files seems to be downloaded extension files with names like `018530b7-a0e1-42af-8227-2df5278250fb`. Every time an extension is updated, a new file is created in `/tmp` and never deleted.

**Comment by sandy081** [comment]
I do not think we are downloading into /tmp folder. VS Code downloads extensions into VS Code application folder. Moreover /tmp folder is managed by the OS and garbage collected by OS. So please check why it is not being garbage collected by the OS.

**Comment by maroony** [comment]
I do not think that the OS has to clean up this. See the related (and closes) issue https://github.com/microsoft/vscode/issues/53110. Perhaps the fix there does not work for remote developing.

**Comment by sandy081** [comment]
These downloads are on remote machine?

**Comment by maroony** [comment]
I use the setting

```json
"remote.downloadExtensionsLocally": true,
```

So extensions get's downloaded locally on the client side and transferred to the remote machine.
